### ISSUE TITLE
Profile switching fails

### ISSUE THREAD

**Issue by jimni** [RoberWare/pytwinkle#2]
I think switching between profiles is not working correctly. 
If we always use `twinkle -c` to start the Twinkle app, it will, by default, use _twinkle.cfg_ (regardless of the _start_user_profile_ variable).

To make it work, we need to invoke something like `twinkle -f 913842 -c` where 913842 is the profile's name (and the config file's name as well). Better yet, Twinkle can run multiple profiles simultaneously like this: `twinkle -f 913842 296346 -c`. And after running it this way, we can switch between profiles with every command: `twinkle --set-profile 913842 --call +123456789`

**Comment by roblamrod** [comment]
Hello @jimni, thank you for your comment.

There is another function instead of `set_account(username, domain, password)`.
You can use `set_account_by_file(path)` in order to import any user profile config file you want.

Any other suggestions are welcome, best regards.

**Comment by jimni** [comment]
I don't think it fixes the issue because the problem is not related to creating config files; it appears in the later stage when you want Twinkle to apply a specific config file, which it will not do so far as you keep using the _start_user_profile_ variable (I think this variable only applies to GUI mode of Twinkle)
